pkg/server/server.go

```diff
@@ -56,9 +57,7 @@ func StartServer(ctx context.Context, config *Config) error {
 		return errors.Wrap(err, "starting kubernetes")
 	}
 
-	if err := startWrangler(ctx, config); err != nil {
-		return errors.Wrap(err, "starting tls server")
-	}
+	go startOnAPIServerReady(ctx, config)
```

Comment:
I appreciate the refactor here but are we potentially losing a start-up/config time error prior to launching the goroutine?

Reply:
NM, I guess a delayed fatal is still a fatal.